pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -34,6 +34,7 @@
     "System.Security.Cryptography.Encoding": "4.0.0",
     "System.Security.Cryptography.Primitives": "4.0.0",
     "System.Security.Cryptography.X509Certificates": "4.1.1",
+    "System.Security.Cryptography.OpenSsl": "4.0.1",
```

Comment:
nit: we generally try to keep these in alphabetical order to we can more easily find packages in the log list.

Reply:
Ok. I will submit a PR to update this json files to fix the sorting.

Reply:
don't worry about it. You can get it next time :)